docs/src/manual.md

```diff
@@ -49,6 +49,60 @@ used instead.
 
 An intercept is included in any GLM by default.
 
+## Categorical variables
+
+Categorical variables will be dummy coded by default if they are non-numeric or if they are [`CategoricalVector`s](https://juliadata.github.io/CategoricalArrays.jl/stable/) within a TableTraits-supporting table (DataFrames, JuliaDB, ColumnTable, etc). Alternatively, you can pass an explicit [contrasts](https://juliastats.github.io/StatsModels.jl/latest/contrasts/) argument if you would like a different contrast coding system or if you are not using DataFrames.
+
+The response (dependent) variable may not be categorical.
+
+Using a `CategoricalVector` constructed with `categorical`:
+
+```jldoctest
+julia> using DataFrames, GLM
+
+julia> data = DataFrame(y = rand(100), x = categorical(repeat([1, 2, 3, 4], 25)));
+
+julia> # It is also possible to convert an existing column to be categorical with categorical!(data, :x)
```

Comment:
Put this in the text instead of in the code?

Reply:
I thought this was clearer with the context of the example. I've removed this line and added a reference to `categorical!` above.